assets/css/style.css

```diff
@@ -1,360 +1,869 @@
-  font-weight: 700;
-  line-height: 44px;
-}
-
-.lang {
-  display: grid;
-  list-style: none;
-  font-family: 'Inter', sans-serif;
-  font-size: 22px;
-  font-weight: 400;
-  line-height: 60px;
-  text-align: center;
-  margin-left: -40px;
-}
-
-.about-me {
-  height: 350px;
-}
-
-.flex-footer {
-  display: flex;
-  padding-top: 70px;
-  width: 375px;
-  height: 700px;
-  align-items: center;
-  justify-content: center;
-  flex-direction: column;
-}
-
-.flex-footer > div {
-  padding: 20px;
-  font-size: 20px;
-  color: black;
-  flex-grow: 1;
-  text-align: center;
-}
-
-.footer-datail {
-  font-family: 'Inter', sans-serif;
-  font-size: 32px;
-  font-weight: 700;
-  line-height: 44px;
-  text-align: center;
-}
-
-input[type=text] {
-  width: 100%;
-  padding: 12px 12px;
-  margin: 8px 0;
+* {
   box-sizing: border-box;
-  border: none;
-  border-bottom: 2px solid #ccc;
 }
 
-input[type=email] {
-  width: 100%;
-  padding: 12px 12px;
-  margin: 8px 0;
-  box-sizing: border-box;
-  border: none;
-  border-bottom: 2px solid #ccc;
-}
-
-textarea {
-  width: 100%;
-  height: 150px;
-  padding: 12px 20px;
-  box-sizing: border-box;
-  border: 2px solid #ccc;
-  border-radius: 4px;
-  background-color: #f8f8f8;
-  font-size: 16px;
-  resize: none;
+.row::after {
+  content: "";
+  clear: both;
+  display: block;
 }
 
-.button {
-  background: #ff6b00;
-  font-family: 'Inter', sans-serif;
-  text-align: center;
-  text-decoration: none;
-  border: none;
-  color: white;
-  padding: 15px 32px;
-  display: inline-block;
-  font-size: 16px;
-  margin: 4px 2px;
-  cursor: pointer;
+[class*="col-"] {
+  float: left;
 }
 
-.button:hover {
-  background-color: #4caf50;
-  color: white;
+/* For desktop: */
+@media only screen and (min-device-width: 768px) {
+  .col-1 { width: 8.33%; }
+  .col-2 { width: 16.66%; }
+  .col-3 { width: 25%; }
+  .col-4 { width: 33.33%; }
+  .col-5 { width: 41.66%; }
+  .col-6 { width: 50%; }
+  .col-7 { width: 58.33%; }
+  .col-8 { width: 66.66%; }
+  .col-9 { width: 75%; }
+  .col-10 { width: 83.33%; }
+  .col-11 { width: 91.66%; }
+  .col-12 { width: 100%; }
+
+  .flex-header {
+    background-image: none !important;
+  }
+
+  .heading-image {
+    float: right;
+    margin-right: -20px;
+  }
+}
+
+/* Desktop device */
+@media only screen and (min-device-width: 768px) {
+  .desktop-hide {
+    display: none;
+  }
+
+  .line {
+    margin-top: 100px;
+  }
+
+  .line2 {
+    height: 2px;
+    color: gray;
+    background-color: gray;
+  }
+
+  .center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 90%;
+  }
+
+  .flex-menu-bar {
+    display: flex;
+    background-color: #3c3a39;
+    width: 100%;
+    height: 40px;
+    flex-direction: column;
+    align-content: center;
+  }
+
+  .social-media {
+    padding-left: 50px;
+    text-align: center;
+  }
+
+  .flex-menu-bar > div {
+    font-size: 30px;
+    color: white;
+    flex-grow: 1;
+  }
+
+  .flex-menu-bar2 {
+    display: flex;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)